frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.unit.spec.tsx

```diff
@@ -74,8 +76,7 @@ async function setup({ user = createMockUser() }) {
   renderWithProviders(
     <Route path="/dashboard/:slug" component={DashboardAppContainer} />,
     {
-      initialRoute: `/dashboard/${TEST_DASHBOARD.id}`,
-      currentUser: user,
```

Comment:
`currentUser` is not actually an option for `renderWithProviders`, there was no error before because the file wasn't in typescript at that time